docs/4.0/utilities/display.md

```diff
@@ -6,20 +6,34 @@ group: utilities
 toc: true
 ---
 
-## Common `display` values
+## How it Works
 
-The [`display` property](https://developer.mozilla.org/en-US/docs/Web/CSS/display) accepts a handful of values and we support many of them with utility classes. We purposefully don't provide every value as a utility, so here's what we support:
+Change the `display` value of elements with our responsive-friendly display utility classes.
 
-- `.d-none`
-- `.d-inline`
-- `.d-inline-block`
-- `.d-block`
-- `.d-table`
-- `.d-table-cell`
-- `.d-flex`
-- `.d-inline-flex`
+The [`display`](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property accepts lots of values, and we support many of them with utility classes. We purposefully don't provide them all.
 
-Put them to use by applying any of the classes to an element of your choice. For example, here's how you could use the inline, block, or inline-block utilities (the same applies to the other classes).
+## Notation
+
+The classes are named using the format 
+* `d-{display}` - for all content
+* `d-{breakpoint}-{display}` - ** for the named breakpoint ( `sm`, `md`, `lg`, `xl`) and above.**
+
+Where *display* is one of:
+
+* `none`
+* `inline`
+* `inline-block`
+* `block`
+* `table`
+* `table-cell`
+* `flex`
+* `inline-flex`
+
+For example, `d-lg-none` sets `display:none` on screens larger than the lg breakpoint.
```

Comment:
- When writing classes, use the `.` in the beginning.
- When writing `property: value;`, use a space and trailing `;`.
- When using the breakpoint shorthand (e.g., "lg"), write it with backticks to render it as code: `lg`.

Reply:
got you. moved the (this breakpoint or wider) emphasis to here

its what confused me at first, the layout/overview/#breakpoints page only mentions `xs` in a comment, so i hadn't noticed it